$OpenBSD: patch-configure,v 1.7 2013/12/01 22:37:36 brad Exp $

Set AUDACIOUS_{CFLAGS,LIBS} only after the end of all plugin checks.
This fixes some of the autoconf checks as the LIBS variable is being
polluted very early on, breaking most of the checks using AC_CHECK_LIB.

--- configure.orig	Fri Nov  1 15:19:49 2013
+++ configure	Fri Nov 29 20:15:57 2013
@@ -8802,8 +8802,6 @@ $as_echo "yes" >&6; }
 
 fi
 
-CPPFLAGS="$CPPFLAGS $AUDACIOUS_CFLAGS"
-LIBS="$LIBS $AUDACIOUS_LIBS"
 
 
 INPUT_PLUGINS="tonegen metronom vtx"
@@ -13319,6 +13317,8 @@ fi
 
 fi
 
+CPPFLAGS="$CPPFLAGS $AUDACIOUS_CFLAGS"
+LIBS="$LIBS $AUDACIOUS_LIBS"
 
 plugindir=`pkg-config audacious --variable=plugin_dir`
 
